ipaserver/install/cainstance.py

```diff
@@ -769,6 +770,11 @@ def __create_ca_agent(self):
             self.basedn)
         conn.add_entry_to_group(user_dn, group_dn, 'uniqueMember')
 
+        if minimum_acme_support():
+            group_dn = DN(('cn', ACME_AGENT_GROUP), ('ou', 'groups'),
+                          self.basedn)
+            conn.add_entry_to_group(user_dn, group_dn, 'uniqueMember')
```

Comment:
`ldap2.add_entry_to_group()` can throw `errors.AlreadyGroupMember` error, so it makes sense to intercept it here for a replica case.

Reply:
This is called in __create_ca_agent() which only executes if not a clone.